src/services/nnas/routes/devices.ts

```diff
@@ -15,4 +15,14 @@ router.get('/@current/status', async (request: express.Request, response: expres
 	}).end());
 });
 
+/**
+ * [PUT]
+ * Replacement for: https://account.nintendo.net/v1/api/devices/@current/inactivate
+ * Description: Used for factory resets, unsure what it exactly does, maybe unlinks the account?
+ */
+router.put('/@current/inactivate', async (request: express.Request, response: express.Response): Promise<void> => {
+	// TODO - Finish this
+	response.status(200).send('');
+});

```

Comment:
Just a note on the comments; what this endpoint does is essentially tell the server to treat the console as if it was fresh from the factory again. It does unlink users (all users), yes, but it would also do things like remove linked purchases and such. You are inactivating the console (device)

Reply:
Ah alright, I'll update the comment.

Reply:
Comment update.